### ISSUE TITLE
Occasional empty pswp__item div

### ISSUE THREAD

**Issue by malinky** [dimsemenov/PhotoSwipe#722]
Sometimes when I open a viewer the last of the 3 pswp__item divs is empty. This means when moving onto that image there is a black screen and an error is thrown that center is undefined in the public method:

updateCurrZoomItem:
(line) _panOffset.x = _currPanBounds.center.x;

It doesn't happen every time but I'm just not sure how to fix this.

Many thanks

**Comment by dimsemenov** [comment]
- Can you repeat the issue on main demo page?
- What browser/OS?
- Can you provide isolated example and step by step instructions on how to reproduce the issue?

**Comment by malinky** [comment]
Hi,

Thanks for you're reply. I've fixed this now. I had amended your code to 
load the viewer from thumbnails and I had set my own data- attributes 
for the image indexes. However, when passing these indexes into the 
options object I hadn't used parseInt() so both my index and the first 
index were trying to open.

Thanks again, great job you've done on the re build of this, I'm going 
to implement responsive image swapping tomorrow.

Thanks

Craig
